subprojects/packagefiles/theora/doc/meson.build

```diff
@@ -0,0 +1,47 @@
+static_docs = files(
+  'color.html',
+  'draft-ietf-avt-rtp-theora-00.txt',
+  'draft-ietf-avt-rtp-theora-00.xml',
+  'vp3-format.txt',
+)
+
+install_data(static_docs)
+
+if doc.allowed()
```

Comment:
Then this checks if doc is enabled or auto, and in the auto case, doxygen can be not-found without the build erroring, but doc.allowed() is still true. It's allowed, even though it didn't get *found*.

Instead, use doxygen_exe.found(), which reports not just whether the build options *allow* it, but whether the build environment *is capable* of it.